### PR TITLE
Astro 5/2 release notes

### PR DESCRIPTION
**Pull request by jwitz**
Tagging PMs who might have a change in this week's release. Also adding @vandyliu for review to confirm one small change.

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -21,6 +21,12 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 **Latest CLI Version**: 1.14.1 ([Release notes](cli/release-notes.md))
 
+## May 2, 2023
+
+### Bug fixes
+
+- Fixed an issue where SSO configurations made through Astronomer support could be overridden by updating the SSO configuration through the Cloud UI.
```
Comment (jwitz) [inline astro/release-notes.md:28]: @vandyliu Is this the right way to describe https://github.com/astronomer/astro/issues/12594?

Comment (vandyliu) [reply astro/release-notes.md:34]: LGTM

**Review by vandyliu — APPROVED** [review]

File: astro/release-notes.md [hunk 2]
```diff
@@ -21,6 +21,18 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 **Latest CLI Version**: 1.14.1 ([Release notes](cli/release-notes.md))
 
+## May 2, 2023
+
+### Receive DAG alerts on Slack or PagerDuty
+
+DAG alerts are a new way to be notified when your DAGs aren't running as expected. Unlike Airflow callbacks and SLAs, Astro DAG alerts require no changes to DAG code and integrate with Slack and PagerDuty.
+
+You can set an alert on any DAG to be notified when the DAG fails or when a task takes longer to run than expected. See [Doc link] for configuration steps.
```
Comment (jwitz) [inline astro/release-notes.md:30]:
```suggestion
You can set an alert on any DAG to be notified when the DAG fails or when a task takes longer to run than expected. See [Astro alerts](alerts.md) for configuration steps.
```

File: astro/release-notes.md [hunk 3]
```diff
@@ -21,6 +21,18 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 **Latest CLI Version**: 1.14.1 ([Release notes](cli/release-notes.md))
 
+## May 2, 2023
+
+### Receive DAG alerts on Slack or PagerDuty
+
+DAG alerts are a new way to be notified when your DAGs aren't running as expected. Unlike Airflow callbacks and SLAs, Astro DAG alerts require no changes to DAG code and integrate with Slack and PagerDuty.
```
Comment (jwitz) [inline astro/release-notes.md:28]:
```suggestion
Astro alerts are a new way to be notified when your DAGs aren't running as expected. Unlike Airflow callbacks and SLAs, Astro alerts require no changes to DAG code and integrate with Slack and PagerDuty.
```

File: astro/release-notes.md [hunk 4]
```diff
@@ -21,6 +21,18 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 **Latest CLI Version**: 1.14.1 ([Release notes](cli/release-notes.md))
 
+## May 2, 2023
+
+### Receive DAG alerts on Slack or PagerDuty
```
Comment (jwitz) [inline astro/release-notes.md:26]:
```suggestion
### Receive Astro alerts on Slack or PagerDuty
```

**Pull request merged**